devito/arch/compiler.py

```diff
@@ -434,6 +434,10 @@ def __init_finalize__(self, **kwargs):
         if platform in [POWER8, POWER9]:
             # -march isn't supported on power architectures, is -mtune needed?
             self.cflags = ['-mcpu=native'] + self.cflags
+        elif platform is Graviton:
```

Comment:
it shuold be `isinstance(platform, Graviton)` no?

Reply:
Fixe